vendor/bat-native-ledger/src/bat/ledger/internal/media/twitch.cc

```diff
@@ -251,12 +245,11 @@ std::string MediaTwitch::GetFaviconUrl(
     return std::string();
   }
 
-  return braveledger_media::ExtractData(publisher_blob,
-    "<figure class=\"tw-avatar tw-avatar--size-36\">"
-    "<div class=\"tw-border-radius-medium tw-overflow-hidden\">"
-    "<img class=\"tw-avatar__img tw-image\" alt=\"" + handle + "\" "
-    "src=\"",
-    "\"");
+  const std::string wrapper = braveledger_media::ExtractData(publisher_blob,
```

Comment:
This won't work if the user is logged in.

The first instance of this pattern when a user is logged in appears for the logged in user's avatar

When not logged in:
![UNADJUSTEDNONRAW_mini_2a9](https://user-images.githubusercontent.com/25326067/59039333-a2ad5b80-8842-11e9-90d4-9939b2d1a697.jpg)


When logged in with my account it shows my avatar:
![UNADJUSTEDNONRAW_mini_2a8](https://user-images.githubusercontent.com/25326067/59039186-5bbf6600-8842-11e9-9610-a1880125dd44.jpg)

Reply:
done